style.css

```diff
@@ -0,0 +1,889 @@
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+
+  .nav-headerfirst li {
+    list-style: none;
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+
+  .LinksumitJoin a {
+    color: white;
+    font-family: 'lato', sans-serif;
+  }
+
+  .nav-headerfirst li a {
+    text-decoration: none;
+    color: white;
+    transition: 1s;
+  }
+
+  .header-second ul li a {
+    text-decoration: none;
+    color: #080000;
+    padding-right: 17px;
+  }
+
+  .nav-headerfirst li a:hover {
+    font-size: x-large;
+  }
+
+  .header-second {
+    display: flex;
+    position: relative;
+    width: 100%;
+    padding-top: 2vh;
+    height: 8vh;
+    background-color: white;
+    justify-content: space-between;
+  }
+
+  .logo {
+    margin-left: 15%;
+  }
+
+  .header-second ul {
+    display: flex;
+  }
+
+  .header-second ul li {
+    list-style: none;
+    font-weight: bold;
+  }
+
+  .content-liCc {
+    width: 130px;
+    border: solid 3px red;
+    margin-right: 50px;
+    justify-self: center;
+  }
+
+  .title-detail {
+    position: absolute;
+    top: 80px;
+    left: 42%;
+    display: flex;
+    justify-content: center;
+    color: hsl(6, 82%, 59%);
+    font-weight: bold;
+    font-size: 12px;
+    margin-top: 10px;
+  }
+
+  .welcom {
+    color: #272a31;
+    position: relative;
+    width: 80%;
+    height: 100vh;
+    margin-top: 30px;
+    margin-left: 15%;
+    z-index: 999;
+    margin-bottom: 0;
+  }
+
+  .welcom-h1 {
+    max-width: 400px;
+    color: hsl(6, 82%, 59%);
+    font-family: 'Lato', sans-serif;
+    line-height: 1.5;
+    font-size: 20px;
+  }
+
+  .welcom-p1 {
+    max-width: 700px;
+    height: auto;
+    color: hsl(6, 82%, 59%);
+    font-family: 'Lato', sans-serif;
+    font-size: 40px;
+    font-weight: bold;
+  }
+
+  .welcom-p2 {
+    max-width: 400px;
+    margin-top: 30px;
+    padding: 15px;
+    margin-bottom: 50px;
+    color: hsl(222, 11%, 17%);
+    font-family: 'Lato', sans-serif;
+    border: solid 5px white;
+    border-radius: 10px 10px 10px 10px;
+    font-size: 15px;
+  }
+
+  .welcom-p3 {
+    margin-bottom: 50px;
+    color: hsl(222, 11%, 17%);
+    font-family: 'Lato', sans-serif;
+    font-size: 30px;
+  }
+
+  .welcom-p3 span {
+    margin-bottom: 50px;
+    color: hsl(222, 11%, 17%);
+    font-family: 'Lato', sans-serif;
+    font-size: 20px;
+    font-weight: bold;
+  }
+
+  .main-project {
+    position: relative;
+    display: flex;
+    background-color: hsl(222, 11%, 17%);
+    width: 100%;
+    padding-top: 20vh;
+    height: 50vh;
+    gap: 5px;
+  }
+
+  .project {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    justify-self: center;
+    background-color: rgb(53, 55, 62);
+    color: #d3d3d3;
+    width: 20%;
+    height: 60%;
+    padding-top: 3vh;
+    padding-bottom: 2vh;
+    font-family: 'Lato', sans-serif;
+    line-height: 1.5;
+    font-size: 10px;
+  }
+
+  .icon-detail {
+    display: flex;
+    justify-content: center;
+  }
```

Comment:
[OPTIONAL]Great job implementing the corrections as suggested by the previous reviewer 👍. Kindly reduce the height of the cards, also It will be better if the space between the main program and the cards is reduced 🥺 

<img width="1470" alt="Screenshot 2023-03-23 at 14 08 38" src="https://user-images.githubusercontent.com/73854999/227214404-b5a767e0-0e3e-4edd-9e77-d2f6d03803ab.png">